### PR TITLE
rgw: Remove unused Parameter in Function RGWConf::init()

### PR DESCRIPTION
**Pull request by C2python**
Signed-off-by: Wen Zhang <zhangwen1@unionpay.com>

### CODE REVIEW

**Comment by C2python** [comment]
Jenkins Retest please

**Comment by C2python** [comment]
@xiexingguo Could you review it? Tks~

**Comment by xiexingguo** [comment]
I am forwarding this to @joscollin

**Comment by joscollin** [comment]
Jenkins retest this please

**Comment by joscollin** [comment]
Jenkins retest this please

File: src/rgw/rgw_common.h [hunk 1]
```diff
@@ -385,7 +385,7 @@ class RGWEnv;
 class RGWConf {
   friend class RGWEnv;
 protected:
```
Comment (joscollin) [inline src/rgw/rgw_common.h]: Drop `protected` as it is not being used and `RGWEnv` is a friend of `RGWConf`.

Comment (C2python) [reply src/rgw/rgw_common.h]: protected has been dropped in patch  #17432

Comment (joscollin) [reply src/rgw/rgw_common.h]: @C2python Yes. As your response got delayed, I have submitted that in my patch. I will continue the review of this. Thanks.

**Comment by joscollin** [comment]
Test Result:

- http://pulpito.front.sepia.ceph.com/jcollin-2017-09-02_13:34:19-rgw-wip-jcollin-testing_09-02-2017-distro-basic-smithi/
- http://pulpito.front.sepia.ceph.com/compare/?branch=wip-jcollin-testing_09-02-2017&suite=rgw

**Pull request merged**